docker/local-test/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
```

Comment:
I'm using this image because it is (I *think*) the same base image that our CI environment uses, and that is what is needed here in this image. In `python-ci.yml` we have `runs-on: ubuntu-18.04`, which I am thinking would imply this. Do you know to the contrary?

Reply:
No you have it right, I forgot that these images are for testing rather than for running.